views/js/review/provider/qtiTestReviewProvider.js

```diff
@@ -139,6 +140,31 @@ define([
 
             this.assetManager = assetManagerFactory();
 
+            // first and second tab will show block and navigate to panel or content
+            const createJumplinks = (container) => {
+                container.on('click', (e) => {
+                    if ($(e.target).hasClass('jumplink')){
+                        e.preventDefault();
+                        $(e.target).blur();
+                        const where = $(e.target).data('area');
```

Comment:
No need to use jQuery here:
```javascript
if (e.target.classList.contains('jumplink')){
    e.preventDefault();
    e.target.blur();
    const where = e.target.dataset.area;
```

At least you should store the jQuery object in a variable to avoid reconstruction on each line

Reply:
removed jQuery